### PR TITLE
webdriverio: add selector property to AsyncSelectors

### LINKED ISSUE #4534: Property "selector" does not exist on type "ElementAsync"
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.12.5
 - **Mode:** Standalone mode
 - **If WDIO Testrunner, running sync/async:** async
 - **Node.js version:** 12.10.0
 - **NPM version:** 6.10.3

**Describe the bug**
WebdriverIO element has property `selector`. `Element` type has access to it. But `ElementAsync` does not.
image

**To Reproduce**
Use `async` mode. Correct type for element in this mode is `ElementAsync`. Try to get `selector` property.

**Expected behavior**
`AsyncElement` should have access to `selector` property

**Additional context**
`packages/webdriverio/webdriverio.d.ts`

```js
// Element commands that should be wrapper with Promise
type ElementPromise = Omit<WebdriverIO.Element,
    'addCommand'
    | '$'
    | '$$'
    | 'selector'
    | 'elementId'
    | 'element-6066-11e4-a52e-4f735466cecf'
    | 'ELEMENT'
>;
...
...
// Element commands wrapper with Promise
type ElementAsync = {
    [K in keyof ElementPromise]:
    (...args: Parameters<ElementPromise[K]>) => Promise<ReturnType<ElementPromise[K]>>;
} & AsyncSelectors;
```

I'm not sure why `selector` is not visible in typescript.

### PR DESCRIPTION
**Pull request by CrispusDH**
## Proposed changes

fix #4534 

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/webdriverio/webdriverio.d.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ type ElementPromise = Omit<WebdriverIO.Element,
 interface AsyncSelectors {
     $: $;
     $$: $$;
+    selector: string
```
Comment (christian-bromann) [inline packages/webdriverio/webdriverio.d.ts]:
```suggestion
    selector: string;
```

Comment (CrispusDH) [reply packages/webdriverio/webdriverio.d.ts]: better to add :)

**Comment by CrispusDH** [comment]
@mgrybyk strange, tests for async has been already there `tests/typings/webdriverio/async.ts`

**Comment by mgrybyk** [comment]
@CrispusDH do you know why these test were passing?

**Comment by mgrybyk** [comment]
hm, I can get selector and elementId property even without the change you've proposed

**Comment by mgrybyk** [comment]
This is because `await $('foo')` return type is `Element` which is `interface Element extends ElementAsync, ElementStatic { }`

**Comment by CrispusDH** [comment]
@mgrybyk so I have use `Element` type instead of `ElementAsync`? Am I understand you correctly?

**Comment by mgrybyk** [comment]
WebdriverIOAsync.Element yes, if it works for you we may close the issue and PR

**Comment by CrispusDH** [comment]
@mgrybyk it works for me, hm, we should document it somewhere. I will think about it. Thanks

**Pull request closed without merging**